.github/workflows/ci.yml

```diff
@@ -204,7 +204,10 @@ jobs:
         uses: actions/upload-artifact@v1
         with:
           name: ${{ github.sha }}
-          path: package-release.zip
+          # Adding the native libraries so the symbol collector craft target can find/upload them
+          path: | 
+            package-release.zip
+            ./src/sentry-java/sentry-android-ndk/build/intermediates/merged_native_libs/release/out/lib/*
```

Comment:
We're caching the build output in `/Plugins`. So the submodule build output does not exist.

Reply:
Added path to be cached too.